### PR TITLE
New package: bats-0.4.0

### PR DESCRIPTION
**Pull request by xelxebar**

### CODE REVIEW

**Comment by Chocimier** [comment]
Fix license and homepage, add noarch, install license with vlicense.

**Comment by xelxebar** [comment]
Oh boy. Putting the license in `homepage` was boneheaded. Thanks.
Same about forgetting to copy over the license file.

Anyway, done!

File: srcpkgs/bats/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'bats'
+pkgname=bats
+version=0.4.0
+revision=1
+depends="bash"
+noarch=yes
```
Comment (maxice8) [inline srcpkgs/bats/template]: noarch= goes directly below revision=

Comment (xelxebar) [reply srcpkgs/bats/template]: Blargh. I'm surprised `xlint` doesn't catch these things. Thanks.

**Pull request merged**